rubi/rubi/data/helpers/query_types.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, List
+
+from eth_typing import ChecksumAddress
+
+
+class SubgraphOffer:
```

Comment:
if we follow the pattern of allowing users to query data without a node connection, we will need to have optional parameters to specify whether the `get_fields` should be formatted or not